lib/my-inject.rb

```diff
@@ -0,0 +1,11 @@
+class Array 
+
+  def my_inject(result, &block)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `block`. If it's necessary, use `_` or `_block` as an argument name to indicate that it won't be used. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)